src/Bridge/OpenAI/DallE.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpLlm\LlmChain\Bridge\OpenAI;
+
+use PhpLlm\LlmChain\Model\Model;
+
+final readonly class DallE implements Model
+{
+    public const DALL_E_2 = 'dall-e-2';
+    public const DALL_E_3 = 'dall-e-3';
+
+    /** @param array<string, mixed> $options The default options for the model usage */
+    public function __construct(
+        private string $version = self::DALL_E_2,
```

Comment:
Should we make version 3 the default?

Reply:
From my PoV both versions are very comparable, i have just decided to use the lesser costful thingy as default. What would, from your opinion, be a profit to make the newer model the default? Within GPT there is also not the o1 model, which is the newest, the default 🤔

Reply:
Fine for me if the costs are that much different